sovryn-token-bridge/federator/src/lib/Federator.js

```diff
@@ -263,7 +303,8 @@ module.exports = class Federator {
         granularity,
         userData,
         transactionId,
-        transactionIdU
+        transactionIdU,
+        signatures
     ) {
```

Comment:
To further save on gas, the backend should limit the number of signatures to the minimum number required before sending to blockchain.